diff --git a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -359,7 +359,8 @@
   endif()
 
   if(   CMAKE_SYSTEM_NAME STREQUAL "Linux"
-     OR CMAKE_SYSTEM_NAME STREQUAL "FreeBSD")
+     OR CMAKE_SYSTEM_NAME STREQUAL "FreeBSD"
+     OR NOCT_TARGET_ZEDBSD)
     list(
       APPEND
       NOCT_ACCEL_BACKEND_SOURCE
@@ -587,7 +588,8 @@
   endif()
 
   if(   CMAKE_SYSTEM_NAME STREQUAL "Linux"
-     OR CMAKE_SYSTEM_NAME STREQUAL "FreeBSD")
+     OR CMAKE_SYSTEM_NAME STREQUAL "FreeBSD"
+     OR NOCT_TARGET_ZEDBSD)
     list(
       APPEND
       NOCT_PUBLIC_CPPFLAGS 
